File: node.js/src/slimmed/application/books/commands/addChapterCommand.ts
//import { BookId } from '#original';
import {
  BookId,
  ChapterContent,
  ChapterTitle,
} from '../../../domain/books/entities';
import { Command } from '../../../infrastructure/commands';

export type AddChapterCommand = Command<
  'AddChapterCommand',
  {
    bookId: BookId;
    chapterTitle: ChapterTitle;
    chapterContent: ChapterContent;
  }
>;

File: java/src/main/java/io/eventdriven/slimdownaggregates/original/application/books/commands/AddChapterCommand.java
package io.eventdriven.slimdownaggregates.original.application.books.commands;


import io.eventdriven.slimdownaggregates.original.domain.books.entities.*;

public record AddChapterCommand(
  BookId bookId,
  ChapterTitle title,
  ChapterContent content
){}
